### PR TITLE
feat: add with-algolia example

### PR DESCRIPTION
**Pull request by notrab**
1. Add Algolia
2. Replace instances of `now.sh` with `vercel.app`

### CODE REVIEW

File: with-algolia/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import algoliasearch from 'algoliasearch/lite';
+import { InstantSearch, SearchBox, Hits } from 'react-instantsearch-dom';
+
+const searchClient = algoliasearch(
+  'HOUK2JPM8O',
+  '7af261a39f13a1d238334a26f5def08b'
```
Comment (ynnoj) [inline with-algolia/pages/index.js:6]: Use the env variable here?

**Review by ynnoj — CHANGES_REQUESTED** [review]
The vercel.app URLs 404 for the existing examples.

**Comment by notrab** [comment]
> The vercel.app URLs 404 for the existing examples.

Oops! I was stuck in a weird rebase I meant to remove these and submit in another PR. Fixed.

File: with-algolia/pages/index.js (hunk 1 above)
Comment (notrab) [reply with-algolia/pages/index.js:6]: I purposely left these in for anyone cloning, and wanting to see some immediate result.

Albeit not very helpful in this instance, but I followed the convention of not using ENV for client-side tokens.

Comment (ynnoj) [reply with-algolia/pages/index.js:6]: > I purposely left these in for anyone cloning, and wanting to see some immediate result.
> 
> Albeit not very helpful in this instance, but I followed the convention of not using ENV for client-side tokens.

Is it worth using the raw token in the API route too then?

Comment (notrab) [reply with-algolia/pages/index.js:6]: Not unless we want unwanted folks changing our Algolia index.

**Review by ynnoj — APPROVED** [review]

**Pull request merged**